componentes/servicos/servicos.scss

```
@@ -0,0 +1,44 @@
+.servicos{
+    &-body{
+        display: flex;
+        justify-content: flex-start;
+        align-items: flex-start;  
+    }
+    &-retangulo{
+        background-color: darkgray;
+        width: 266px;
+        height: 264px;
+        margin-left: 50px; 
+    }
+    &-info{
+        margin-left: 100px;  
+    }
+    &-info__content{
+        display: flex;
+        justify-content: flex-start;
+        align-items: flex-start;
+        margin-bottom: 40px;  
+    }
+    &-info__content-title{
+        text-transform: uppercase;
+        margin: 0;
+    }
```

O que voce acha de alinhar essas `filhos` de `info`?

```suggestion
    &-info{
        margin-left: 100px;  
        
        &__content{
          display: flex;
          justify-content: flex-start;
          align-items: flex-start;
          margin-bottom: 40px; 
          
          &-title {
            text-transform: uppercase;
            margin: 0;
         }
      }
    }
```